### ISSUE TITLE
Please help update the Clients list

### ISSUE THREAD

**Issue by mattl** [foocorp/hacienda#38]
There are lots of clients out there which supported Libre.fm at some point. 

Let's get that list and update it!

https://github.com/foocorp/hacienda/wiki/Clients is a wiki page you can edit. 

Please volunteer below to take a few clients from the old (linked) client codes page and put it into the new page. Also make sure any clients you use are well represented. 

I've added Web Scrobbler as it's the thing I use the most lately. I also use DeadBeef, so I'll volunteer to add that to the list.

**Comment by mattl** [comment]
@foocorp/beta-testers this is a great less-technical task if anyone's looking to jump on.

* I'll add Deadbeef to the list.

**Comment by mattl** [comment]
@ millette Can you please fix the Clients page? You've removed the useful columns and added one we don't need.

**Comment by millette** [comment]
Why was the list mostly deleted? <https://github.com/foocorp/hacienda/wiki/Clients/_compare/dff45187ad8eae7f6ea7cf2fd2a723c851adeed6...913cdde0dba5f02629dcaec012ebfe9653b83f49>

**Comment by mattl** [comment]
@millette look at the history. The page was replaced with half the data removed. I restored it back to the last time it was good.

**Comment by mattl** [comment]
Looks like some more work has been done but edits made by @janerationx were removed?

**Comment by janerationx** [comment]
I added Foobar2000 player last night. And yep, it's been removed. No idea why. 
 Should i just re-add it?

**Comment by mattl** [comment]
I also don’t know why we inherited a new column. I think I’m going to tweak the wiki. I’ll bring it back later

**Comment by mattl** [comment]
Okay, I've moved the wiki to read only and made a new repo, https://github.com/foocorp/wiki.libre.fm -- please make any changes there and submit pull requests.

@millette and @janerationx can just edit stuff without needing to do this.

**Comment by inan-hira** [comment]
@mattl I cannot access wiki.libre.fm. Which wiki software does it use? I can see it supports markdown, but is there a specific flavour of markdown we should use?

**Comment by mattl** [comment]
There’s no website yet, this will get hopefully copied back to the hacienda project’s wiki repo.

**Comment by mattl** [comment]
@foocorp/beta-testers I've added documentation on how to contribute to the wiki.

https://github.com/foocorp/documentation/wiki/Contributing-to-the-wiki

**Comment by millette** [comment]
[ListenBrainz has a list of clients](https://listenbrainz.org/add-data/). A few more are listed here: <https://www.reddit.com/r/lastfm/comments/18s9j4b/comment/kf7t291/>. There used to be <https://build.last.fm> but now there are only archives: <https://archive.is/Rgi7X>.

**Comment by mattl** [comment]
The client codes page is a good place to find old clients that may not be around anymore.

I added stub pages for all the players listed. Please add links and let us know if you're using those players.

**Comment by mattl** [comment]
A suggested workflow would be something like this.

* Check the old client codes page for a potential client
* Install it and see if it works
* If it works, [add it to the clients page](https://github.com/foocorp/documentation/wiki/Contributing-to-the-wiki) -- and if it doesn't, update the client codes page and move it to the bottom (under legacy clients)
* Make a short page about it like the one for [DeadBeeF](https://github.com/foocorp/hacienda/wiki/DeaDBeeF)

**Comment by themidnightmaniac** [comment]
Planning on adding a free/open source column?

**Comment by mattl** [comment]
@themidnightmaniac I'm not sure it's relevant or clear. Are people going to switch to another player because the scrobbling plugin isn't open source?

**Comment by themidnightmaniac** [comment]
@mattl I see. And most standalone scrobblers are open source too.

**Comment by mattl** [comment]
Yeah that’s worth mentioning on the pages for each plugin or app. 

I’m seeing a lot of proprietary apps for iOS scrobbling fwiw.

**Comment by themidnightmaniac** [comment]
Do I need to fork the repo and open a pr to add a client? I'm not seeing an edit button in the wiki.

**Comment by mattl** [comment]
Yes. https://github.com/foocorp/documentation/wiki/Contributing-to-the-wiki

**Comment by themidnightmaniac** [comment]
Ok thx!
